tutorials/text_processing/ITN_with_Thutmose_Tagger.ipynb

```diff
@@ -0,0 +1,1047 @@
+{
+  "cells": [
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "VFOY_ljrReXk"
+      },
+      "outputs": [],
+      "source": [
+        "\"\"\"\n",
+        "You can run either this notebook locally (if you have all the dependencies and a GPU) or on Google Colab.\n",
+        "\n",
+        "Instructions for setting up Colab are as follows:\n",
+        "1. Open a new Python 3 notebook.\n",
+        "2. Import this notebook from GitHub (File -> Upload Notebook -> \"GITHUB\" tab -> copy/paste GitHub URL)\n",
+        "3. Connect to an instance with a GPU (Runtime -> Change runtime type -> select \"GPU\" for hardware accelerator)\n",
+        "4. Run this cell to set up dependencies.\n",
+        "\"\"\"\n",
+        "\n",
+        "import os\n",
+        "\n",
+        "# install NeMo\n",
+        "BRANCH = 'main'\n",
+        "\n",
+        "GITHUB_ACCOUNT = 'NVIDIA'\n",
```

Comment:
could you add a comment that it's either nvidia or fork name

Reply:
done